frontend/src/components/molecules/MatchItem/MatchItem.tsx

```diff
@@ -14,23 +14,23 @@ import MatchWinnerDeterminationDialog from "../MatchWinnerDeterminationDialog/Ma
 import {MatchStatus} from "../../atoms/MatchStatus";
 
 export type MatchItemProps = {
-   onClickTeam: (matchId: string, teamId: string) => void,
-   matchNumber: number | undefined,
-   matchId: string | undefined,
    level: number | undefined,
+   matchId: string | undefined,
```

Comment:
Ciagle mnie to zastanawia, jak moze byc match bez matchId? czy bez level / matchNumbera.